_ml-commons-plugin/algorithms.md

```diff
@@ -412,23 +423,27 @@ The Localization algorithm can only be executed directly. Therefore, it cannot b
 
 A classification algorithm, logistic regression models the probability of a discrete outcome given an input variable. In ML Commons, these classifications include both binary and multi-class. The most common is the binary classification, which takes two values, such as "true/false" or "yes/no", and predicts the outcome based on the values specified. Alternatively, a multi-class output can categorize different inputs based on type. This makes logistic regression most useful for situations where you are trying to determine how your inputs fit best into a specified category. 
 
+**Optimisers supported:** `SIMPLE_SGD`, `LINEAR_DECAY_SGD`, `SQRT_DECAY_SGD`, `ADA_GRAD`, `ADA_DELTA`, `ADAM`, and `RMS_PROP`.  
```

Comment:
Could we add a description for each optimizer, objective, and momentum type so the user can choose appropriately?